Week3/homework-midy/lottery.js

```diff
@@ -0,0 +1,35 @@
+function threeFive(startIndex, endIndex, three, five) {
+  const numbers = [];
+
+  const startParsed = parseInt(startIndex, 10);
+  const endParsed = parseInt(endIndex, 10);
+
+  const random = (start, end) => {
+    for (let i = 0; i < 6; i += 1) {
+      numbers.push(Math.floor(Math.random() * (end - start + 1) + start));
+    }
+    return numbers;
+  };
+
+  random(startParsed, endParsed);
+  numbers.forEach(number => {
+    if (number % 3 === 0 && number % 5 === 0) {
+      console.log(`${number} can be divided by both 3 and 5`);
```

Comment:
According to the exercise you shouldn't console.log here
> Both functions should be called if the array value is divisible by both 3 and 5.

Reply:
Oops.